sssoja/tv-bland
# removes Nunito font

### What changes have you made?
- Initially fixed the pathname for Nunito so that `GlobalStyles` was pointing at `url(/fonts/Nunito/Nunito-ExtraBold.ttf)` 
- Upon solving, `Sans-serif` looked closer to the intended outcome than `Nunito` 

Nunito: 
<img width="298" alt="Screenshot 2020-12-02 at 13 19 55" src="https://user-images.githubusercontent.com/53219789/100885834-64ced900-34b3-11eb-9102-b8914a9d1a1d.png">

Sans-serif: 
<img width="298" alt="Screenshot 2020-12-02 at 13 19 59" src="https://user-images.githubusercontent.com/53219789/100885849-68626000-34b3-11eb-9e9e-c3a2c0ca0862.png">

- I removed `Nunito` entirely and tidied up `GlobalStyles` and `theme` so that the primary font is now `"Sans-serif" and font-weights are used to differentiate between HTML tags

### Which issue(s) does this PR fix?
Fixes #14 

### Screenshots (if there are design changes)
No design changes as the fallback font family is now the primary 

### How to test
When you open `localhost:3000` go to the console and check that both the homepage and the show page are displaying in `sans-serif`
